components/blitz/src/omero/cmd/basic/DoAllI.java

```diff
@@ -259,12 +259,7 @@ public void init(Helper helper) {
             Map<String, String> allgroups = new HashMap<String, String>();
             allgroups.put("omero.group", "-1");
             ctx.publishMessage(new ContextMessage.Push(this, allgroups));
-            try {
-                // Process within -1 block.
-                GraphUtil.combineFacadeRequests(this.requests);
-            } finally {
-                ctx.publishMessage(new ContextMessage.Pop(this, allgroups));
-            }
+            ctx.publishMessage(new ContextMessage.Pop(this, allgroups));
```

Comment:
Push/Pop were there to all `combineFacadeRequests` do its work. Since it's no longer doing anything, they aren't needed.